app/components/feeds/FeedCard.module.css

```
@@ -0,0 +1,70 @@
+.clickableCard {
```

Stylelint (17.2.0)

[error] Expected class selector ".clickableCard" to be kebab-case (selector-class-pattern)

(selector-class-pattern)